Source/WebKit/UIProcess/API/glib/WebKitWebView.cpp

```diff
@@ -165,7 +165,9 @@ enum {
     RUN_FILE_CHOOSER,
 
     CONTEXT_MENU,
+#if PLATFORM(GTK)
     CONTEXT_MENU_DISMISSED,
+#endif
```

Comment:
hmm, this an api break even when this never worked for wpe.

Reply:
I can add it back if that's simpler.

Reply:
Yeah you can't ever remove signals from the public API. You could deprecate it for WPE, though. This would require moving the doc comment to a header file; unifdef will then respect `#if PLATFORM(GTK)` or `#if PLATFORM(WPE)` guards when generating the actual header that gets installed and scanned for doc comments.

Reply:
You would add:

```
Deprecated: 2.48, WPE WebKit does not emit this signal
```

to the bottom of the doc comment, and then also change:

```
#if PLATFORM(GTK)
G_SIGNAL_RUN_LAST,
#else
static_cast<GSignalFlags>(G_SIGNAL_RUN_LAST | G_SIGNAL_DEPRECATED),
#endif
```

Reply:
Thanks for laying it out so clearly for me, would have taken me much longer otherwise!